collections/src/main.rs

```diff
@@ -18,7 +23,7 @@ fn list_med(list: &[i32]) -> ListResult {
         for v in list {
             sum += *v;
         }
-        sum / len as i32
+        sum / len as i32 // TODO
```

Comment:
Also, avoid casting (if possible) in favor of `From`/`TryFrom` implementations.

Reply:
How this can be done in this case? ```i32::from``` doesn't seem to like ```usize```.

Reply:
@amadare42 use https://std.rs for API exploration.

The problem that `usize` is not safely convertible to `i32`, because `usize` values may overflow the values in `i32`. That's why there is not direct `From` conversion, but there is a [fallible `TryFrom` conversion instead](https://doc.rust-lang.org/stable/std/primitive.i32.html#impl-TryFrom%3Cusize%3E).